examples/nirfmxbluetooth/txp-basic.py

```diff
@@ -70,236 +71,232 @@
+            instrument=instr,
+            selector_string="",
+            result_name="",
         )
     )
+    check_for_warning(initiate_response, instr)
 
     ### Fetch Results ###
 
-    txp_fetch_powers_response = raise_if_error(
-        client.TXPFetchPowers(
-            nirfmxbluetooth_types.TXPFetchPowersRequest(
-                instrument=instr, selector_string="", timeout=10.0
-            )
+    txp_fetch_powers_response = client.TXPFetchPowers(
+        nirfmxbluetooth_types.TXPFetchPowersRequest(
+            instrument=instr,
+            selector_string="",
+            timeout=10.0,
         )
     )
+    check_for_warning(txp_fetch_powers_response, instr)
     average_power_mean = txp_fetch_powers_response.average_power_mean
     average_power_maximum = txp_fetch_powers_response.average_power_maximum
     average_power_minimum = txp_fetch_powers_response.average_power_minimum
     peak_to_average_power_ratio_maximum = (
         txp_fetch_powers_response.peak_to_average_power_ratio_maximum
     )
-    txp_fetch_edr_powers_response = raise_if_error(
-        client.TXPFetchEDRPowers(
-            nirfmxbluetooth_types.TXPFetchEDRPowersRequest(
-                instrument=instr, selector_string="", timeout=10.0
-            )
+
+    txp_fetch_edr_powers_response = client.TXPFetchEDRPowers(
+        nirfmxbluetooth_types.TXPFetchEDRPowersRequest(
+            instrument=instr,
+            selector_string="",
+            timeout=10.0,
         )
     )
+    check_for_warning(txp_fetch_edr_powers_response, instr)
     edrgfsk_average_power_mean = txp_fetch_edr_powers_response.edr_gfsk_average_power_mean
     edrdpsk_average_power_mean = txp_fetch_edr_powers_response.edr_dpsk_average_power_mean
     edr_dpsk_gfsk_average_power_ratio_mean = (
         txp_fetch_edr_powers_response.edr_dpsk_gfsk_average_power_ratio_mean
     )
-    txp_fetch_lecte_reference_period_powers_response = raise_if_error(
-        client.TXPFetchLECTEReferencePeriodPowers(
-            nirfmxbluetooth_types.TXPFetchLECTEReferencePeriodPowersRequest(
-                instrument=instr, selector_string="", timeout=10.0
-            )
+
+    txp_fetch_lecte_reference_period_powers_response = client.TXPFetchLECTEReferencePeriodPowers(
+        nirfmxbluetooth_types.TXPFetchLECTEReferencePeriodPowersRequest(
+            instrument=instr,
+            selector_string="",
+            timeout=10.0,
         )
     )
+    check_for_warning(txp_fetch_lecte_reference_period_powers_response, instr)
     reference_period_average_power_mean = (
         txp_fetch_lecte_reference_period_powers_response.reference_period_average_power_mean
     )
     reference_period_peak_absolute_power_deviation_maximum = (
         txp_fetch_lecte_reference_period_powers_response.reference_period_peak_absolute_power_deviation_maximum
     )
 
-    txp_fetch_lecte_transmit_slot_powers_array_response = raise_if_error(
+    txp_fetch_lecte_transmit_slot_powers_array_response = (
```

Comment:
Nit: I know this is existing code, but that is _way too long_ of a variable name for me to sit by idly. If you remove the word `array` from it, it's still obvious what it's referring to. Can we at least get it short enough so the method name isn't split to the next line?

Reply:
removed array